lib/ueberauth/strategy/slack/oauth.ex

```diff
@@ -34,11 +37,14 @@ defmodule Ueberauth.Strategy.Slack.OAuth do
   end
 
   def get_token!(params \\ [], options \\ %{}) do
-    headers        = Dict.get(options, :headers, [])
-    options        = Dict.get(options, :options, [])
-    client_options = Dict.get(options, :client_options, [])
-    client = OAuth2.Client.get_token!(client(client_options), params, headers, options)
-    client.token
+    %{headers: headers, options: %{ options: opts } } = options
```

Comment:
Can you please remove extra whitespace around the `{`, we want to try to be consistent 👍